src/js/views/views/HistoryAll.js

```diff
@@ -44,12 +54,40 @@ const HistoryAll = ({seasons = []}) => {
             countText = null;
         }
 
+        /*
```

Comment:
Get rid of all these comments, IMHO they code is readable enough without them due to the variable names, so the comments just add clutter.